### ISSUE TITLE
Single line updates not showing

### ISSUE THREAD

**Issue by EvertEt** [mthenw/frontail#231]
When using the following python script and commands, no updates are printed (no progress dots)
```
import sys
from time import sleep

sys.stdout.write("starting\n")
while True:
    sys.stdout.write(".")
    sleep(3)
```

```
python -u progress.py > test.log
```

```
frontail test.log
```

Is there a way to get these updates to match the `tail -F` command which does show and update the progress dots line?

**Comment by mthenw** [comment]
yeah, the thing is that this script doesn't produce new lines. Frontal doesn't react to those changes.

**Comment by EvertEt** [comment]
Thank you for your response. Is this on purpose? Would there be a way or option to enable this to get closer to the real tail behaviour?

**Comment by mthenw** [comment]
It's not on purpose. Assuming that there is always a new line simplifies the implementation. I've never considered changing that behaviour as it would require lots of changes (I guess).
